tls/s2n_cipher_suites.c

```diff
@@ -510,6 +510,18 @@ const struct s2n_cipher_preferences cipher_preferences_test_all_fips = {
     .minimum_protocol_version = S2N_TLS10
 };
 
+/**
```

Comment:
What do you think about moving these macros to https://github.com/awslabs/s2n/blob/master/crypto/s2n_openssl.h and possibly reusing the existing ` S2N_OPENSSL_VERSION_AT_LEAST` for the version checks here?

Reply:
That sounds like a good idea. I'll give it a go.

Reply:
done